### ISSUE TITLE
MMSE plots

### ISSUE THREAD

**Issue by andkov** [IALSA/ialsa-2016-time-sensitivity#1]
@GracielaMuniz @ampiccinin @annierobi

## Linear models
![](https://raw.githubusercontent.com/IALSA/ialsa-2016-time-sensitivity/d2e3713ad3e074f5676aa2d56ad402adaa042561/reports/model-grapher/graphs-1/L-12345.png)
![](https://raw.githubusercontent.com/IALSA/ialsa-2016-time-sensitivity/d2e3713ad3e074f5676aa2d56ad402adaa042561/reports/model-grapher/graphs-1/L-1234.png)
![](https://raw.githubusercontent.com/IALSA/ialsa-2016-time-sensitivity/d2e3713ad3e074f5676aa2d56ad402adaa042561/reports/model-grapher/graphs-1/L-123.png)
![](https://raw.githubusercontent.com/IALSA/ialsa-2016-time-sensitivity/d2e3713ad3e074f5676aa2d56ad402adaa042561/reports/model-grapher/graphs-1/L-135.png)

## Quadratic models
![](https://raw.githubusercontent.com/IALSA/ialsa-2016-time-sensitivity/d2e3713ad3e074f5676aa2d56ad402adaa042561/reports/model-grapher/graphs-1/Q-12345.png)
![](https://raw.githubusercontent.com/IALSA/ialsa-2016-time-sensitivity/d2e3713ad3e074f5676aa2d56ad402adaa042561/reports/model-grapher/graphs-1/Q-1234.png)
![](https://raw.githubusercontent.com/IALSA/ialsa-2016-time-sensitivity/d2e3713ad3e074f5676aa2d56ad402adaa042561/reports/model-grapher/graphs-1/Q-123.png)
![](https://raw.githubusercontent.com/IALSA/ialsa-2016-time-sensitivity/d2e3713ad3e074f5676aa2d56ad402adaa042561/reports/model-grapher/graphs-1/Q-135.png)

**Comment by ampiccinin** [comment]
@andkov @GracielaMuniz @annierobi

Odd that the quadratic don't seem much less linear, but that the estimates go above 30, where they don't with the linear. 

Were these done in Mplus? Was tobit used? For quadratic, but not linear, perhaps? 

Nice graphs!

**Comment by andkov** [comment]
Yes, all these were done in Mplus and processed through Portland-like pipeline. I didn't upload the outputs because they were too many,but in retrospect it appears an error of judgement. I'll push the Mplus files underlying these graphs later today when I get to my laptop - it's no trouble.

**Comment by annierobi** [comment]
Tobit was not used for these models.  Unless new models were run that I am
unaware of. The models in dropbox do not use tobit.

**Comment by andkov** [comment]
The models on Dropbox that I used as guides for automation did not use Tobit. The models behinds these graphs also don't use it.

**Comment by andkov** [comment]
@ampiccinin 
The output files have been uploaded to https://github.com/IALSA/ialsa-2016-time-sensitivity/tree/master/output/studies/octo/
